src/database/seeds/0002-update-canonical_fields.sql

```diff
@@ -0,0 +1,188 @@
+('Proposal Collects New Data from Individuals', 'proposal_collects_new_data_from_individuals', '{ "type": "boolean" }' ),
+('Proposal Collects Personal Data', 'proposal_collects_personal_data', '{ "type": "boolean" }' ),
+('Proposal Collects New or Personal Data Explanation', 'proposal_collects_new_or_personal_data_explanation', '{ "type": "string" }' ),
+('Proposal Research Will be Published', 'proposal_research_will_be_published', '{ "type": "boolean" }' ),
+('Organization has Human Subjects Research Policy', 'organization_has_human_subjects_research_policy', '{ "type": "boolean" }' ),
+('Proposal Research Involves Minors', 'proposal_research_involves_minors', '{ "type": "boolean" }' ),
+('Organization has Child Protection Policy', 'organization_has_child_protection_policy', '{ "type": "boolean" }' ),
+('Proposal Funds Fiscal Agent Duties Research or Conference', 'proposal_funds_fiscal_agent_duties_research_or_conference', '{ "type": "boolean" }' ),
+('Proposal Funds Fiscal Agent Duties Research or Conference Explanation', 'proposal_funds_fiscal_agent_duties_research_or_conference_explanation', '{ "type": "string" }' ),
+('Additional Information From Applicant', 'additional_information_from_applicant', '{ "type": "string" }' ),
+('Organization Articles of Incorporation Document', 'organization_articles_of_incorporation_document', '{ "type": "uri" }' ),
+('Organization Child Protection Policy Document', 'organization_child_protection_policy_document', '{ "type": "uri" }' ),
+('Organization Conflict of Interest Policy Document', 'organization_conflict_of_interest_policy_document', '{ "type": "uri" }' ),
+('Organization Human Subjects Research Policy Document', 'organization_human_subjects_research_policy_document', '{ "type": "uri" }' ),
+('Organization IRS Tax Letter Document', 'organization_irs_tax_letter_document', '{ "type": "uri" }' ),
+('Organization Latest Unaudited Financial Statement Document', 'organization_latest_unaudited_financial_statement_document', '{ "type": "uri" }' ),
+('Organization Charter Document', 'organization_charter_document', '{ "type": "uri" }' ),
+('Organization Policy Regarding Data Gathering from Children Document', 'organization_policy_regarding_data_gathering_from_children_document', '{ "type": "uri" }' ),
+('Organization Whistleblower Policy Document', 'organization_whistleblower_policy_document', '{ "type": "uri" }' ),
+('Organization Latitude EPSG:4326', 'organization_latitude_epsg_4326', '{ "type": "number" }' ),
+('Organization Longitude EPSG:4326', 'organization_longitude_epsg_4326', '{ "type": "number" }' ),
+('Organization Neighborhood', 'organization_neighborhood', '{ "type": "string" }' ),
+('Proposal Abstract', 'proposal_abstract', '{ "type": "string" }' ),
+('Proposal Purpose Statement', 'proposal_purpose_statement', '{ "type": "string" }' ),
+('Proposal Agreement with Fiscal Agent Docume', 'proposal_agreement_with_fiscal_agent_document', '{ "type": "uri" }' );
```

Comment:
Looks like the label got clipped here:

```suggestion
('Proposal Agreement with Fiscal Agent Document', 'proposal_agreement_with_fiscal_agent_document', '{ "type": "uri" }' );
```

Reply:
Nice catch. Yes, this needs to be fixed in another set of changes.

Reply:
It looks like this still needs improvement because I see it in the prod base fields. I created https://github.com/PhilanthropyDataCommons/service/issues/605 to track this one.